sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ReceiveMessageAndSettleAsyncSample.java

```diff
@@ -9,10 +9,10 @@
 
 /**
  * Sample demonstrates how to receive an {@link ServiceBusReceivedMessage} from an Azure Service Bus Queue and settle
- * it. Settling of message include {@link ServiceBusReceiverAsyncClient#complete(String) complete()}, {@link
- * ServiceBusReceiverAsyncClient#defer(String) defer()},
- * {@link ServiceBusReceiverAsyncClient#abandon(String) abandon}, or
- * {@link ServiceBusReceiverAsyncClient#deadLetter(String) dead-letter} a message.
+ * it. Settling of message include {@link ServiceBusReceiverAsyncClient#complete(ServiceBusReceivedMessage)} complete()}
```

Comment:
These javadocs are not correct.

Reply:
you mean it has extra `}` . I removed it.